src/D2L.CodeStyle.Analyzers/ApiUsage/DangerousMemberUsages/DangerousMemberUsagesAnalyzer.cs

```diff
@@ -18,6 +19,9 @@ internal sealed class DangerousMemberUsagesAnalyzer : DiagnosticAnalyzer {
 		private const string DangerousPropertyAuditedAttributeFullName = "D2L.CodeStyle.Annotations.DangerousPropertyUsage+AuditedAttribute";
 		private const string DangerousPropertyUnauditedAttributeFullName = "D2L.CodeStyle.Annotations.DangerousPropertyUsage+UnauditedAttribute";
 
+		private const string DangerousMethodFullName = "D2L.CodeStyle.Annotations.Objects+DangerousMethod";
+		private const string DangerousPropertyFullName = "D2L.CodeStyle.Annotations.Objects+DangerousProperty";

```

Comment:
I'm not feeling confident these belong on `Objects`. Perhaps `DangerousMethodUsage+DeclareAttribute` or just `DangerousMethodAttribute`?